src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -112,6 +121,7 @@ const typeDefs = `
 
     metacardsByTag(tag: String!, settings: QuerySettingsInput): QueryResponse
     metacardsById(ids: [ID]!, settings: QuerySettingsInput): [QueryResponse]
+    metacardSharingAttributes(id: ID, settings: QuerySettingsInput) : Accessors
```

Comment:
I'm not sure we need to have a new resolver for this. I would recommend moving this logic to a react hook on the frontend